blaze/server/serialization.py

```diff
@@ -2,13 +2,65 @@
 from functools import partial
 import json as json_module
 
+from datashape.predicates import iscollection, istabular, isscalar
+from odo import odo
+import pandas as pd
+from pandas.compat import BytesIO
+from pandas.io.packers import (
+    unpack as msgpack_unpack,
+    decode as pd_msgpack_object_hook,
+    encode as pd_msgpack_default,
+)
 import pandas.msgpack as msgpack_module
+from toolz import identity
 
-from ..compatibility import pickle as pickle_module, unicode
+from ..compatibility import pickle as pickle_module, unicode, PY2
+from ..interactive import coerce_scalar
 from ..utils import json_dumps, object_hook
 
 
-SerializationFormat = namedtuple('SerializationFormat', 'name loads dumps')
+_serializationformat = namedtuple(
```

Comment:
In the original implementation we only had `loads` and `dumps`. We used tuple indexing instead of named access. We have since changed and now always use the named fields. I think we can get rid of this entirely and just subclass `object` without slots.